### PR TITLE
dev-python/notebook: Version bump to 4.3.0

### PR DESCRIPTION
**Pull request by marbre**
@SoapGentoo Simple copy of the previous ebuild. Tests pass for python 2.7 and 3.4, with commit 733fbcdcf08b74153bb6c3786e8f79ba031cb339 reverted, with breaks my test system (see bug [524322](https://bugs.gentoo.org/show_bug.cgi?id=524322)).

### CODE REVIEW

File: dev-python/notebook/notebook-4.3.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+python_test() {
+	nosetests \
+		--verbosity=3 \
+		notebook || die
+}
+
+python_install() {
+	distutils-r1_python_install
+
+	ln -sf \
```
Comment (SoapGentoo) [inline dev-python/notebook/notebook-4.3.0.ebuild]: have you tried using dosym here instead? More Gentoo-ish

File: dev-python/notebook/notebook-4.3.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+}
+
+python_install_all() {
+	use doc && HTML_DOCS=( docs/build/html/. )
+	distutils-r1_python_install_all
+}
+
+pkg_preinst() {
+	# remove old mathjax folder if present
+	rm -rf "${EROOT}"/usr/lib*/python*/site-packages/notebook/static/components/MathJax || die
```
Comment (SoapGentoo) [inline dev-python/notebook/notebook-4.3.0.ebuild]: double-slashed, make this `"${EROOT%/}"`

Comment (marbre) [inline dev-python/notebook/notebook-4.3.0.ebuild]: Done.

File: dev-python/notebook/notebook-4.3.0.ebuild (hunk 1 above)
Comment (marbre) [inline dev-python/notebook/notebook-4.3.0.ebuild]: @SoapGentoo So, do we leave it as is or do we have to change this?

**Pull request closed without merging**